src/V3LinkDot.cpp

```diff
@@ -1123,6 +1124,48 @@ class LinkDotFindVisitor final : public VNVisitor {
             m_ftaskp = nullptr;
         }
     }
+    void visit(AstClocking* nodep) override {
+        VL_RESTORER(m_clockingp);
+        m_clockingp = nodep;
+        iterate(nodep->sensesp());
+        iterateAndNextNull(nodep->itemsp());
+        if (nodep->name().empty()) return;
```

Comment:
I believe you cannot reference clockvars in unnamed clocking blocks. So there is no need to go in there (and it would probably screw up the real vars).